pkg/api/handlers/libpod/volumes.go

```diff
@@ -47,9 +48,19 @@ func CreateVolume(w http.ResponseWriter, r *http.Request) {
 	if len(input.Driver) > 0 {
 		volumeOptions = append(volumeOptions, libpod.WithVolumeDriver(input.Driver))
 	}
-	if len(input.Label) > 0 {
-		volumeOptions = append(volumeOptions, libpod.WithVolumeLabels(input.Label))
+
+	// Label provided for compatibility.
```

Comment:
Can we possibly error if the user provides both? If we allow both to be specified we get into weird edge-cases where both specify different values for the same label.